packages/pangea-sdk/pangea/request.py

```diff
@@ -49,12 +50,25 @@ def __init__(
 
         # Custom headers
         self._extra_headers = {}
-
         self.session: requests.Session = self._init_session()
 
+        self.logger = logging.getLogger(self.service)
+        handler = TimedRotatingFileHandler(
+            filename="pange_sdk.log", when="D", interval=1, backupCount=90, encoding="utf-8", delay=False
+        )
+        formatter = logging.Formatter(
+            fmt="{{'time': '%(asctime)s.%(msecs)03d', 'name': '%(name)s', 'level': '%(levelname)s',  'message': %(message)s }}",
+            datefmt="%Y-%m-%d %H:%M:%S",
+        )
+        handler.setFormatter(formatter)
+        self.logger.addHandler(handler)
+
     def __del__(self):
         self.session.close()
 
+    def set_logger_level(self, level: int = logging.INFO):
```

Comment:
if we do it the way Govind is suggesting, the user will be able to control this on their own without relying on our code